esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -220,7 +220,7 @@ def _regrid_unstructured_to_rectilinear__perform(src_cube, regrid_info, mdtol):
     if len(grid_x.shape) == 1:
         chunk_shape = (len(grid_x.points), len(grid_y.points))
     else:
-        chunk_shape = grid_x.shape
+        chunk_shape = grid_x.shape[::-1]
```

Comment:
I can't find where this is tested that it is preserving laziness?

Reply:
As discussed offline, this change doesn't "preserve laziness", but rather that it is a fix required as the lazy data was failing.